python/GafferTest/ContextTest.py

```diff
@@ -684,6 +684,66 @@ def testOmitCanceller( self ) :
 		context3 = Gaffer.Context( context1, omitCanceller = False )
 		self.assertIsNotNone( context3.canceller() )
 
+	@staticmethod
+	def collisionCountParallelHelper( seed, mode, chunk, countList ):
+		countList[0] = GafferTest.countContextHash32Collisions( 2**20, mode, chunk, seed )
+
+	@unittest.skipIf( GafferTest.inCI(), "Too expensive to run on CI, and has a small chance of failure because it is statistical" )
```

Comment:
This is too expensive to run as a matter of course. I rely on being able to run the full test suite for a module fairly regularly as I work, and this kills that ability (a few recent additions are pushing the boundaries already). Perhaps we should extend the `gaffer test -performanceOnly` idea to something like `gaffer test -type all|standard|performance|veryVerySlow`?

Reply:
> I guess the quick variant would be setting skipIf( True ) and leaving it up to anyone who is doing hash experimentation in the future to manually turn it on?

I'll buy that for now.

> If we do take a look at the args to gaffer test, one that I should add is "always print performance times".

Sounds good. Let's aim to tackle these together in the not-too-distant future.